pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.github.invictum</groupId>
     <artifactId>serenity-reportportal-integration</artifactId>
-    <version>1.6.0-SNAPSHOT</version>
+    <version>1.7.0-SNAPSHOT</version>
```

Comment:
Nit: Do we need to update it manually?
It is automated via release plugin and it is minor increment since Serenity update is also minor.